src/components/Dialog.tsx

```diff
@@ -339,7 +348,12 @@ export default function Dialog({ color, children, onClose, forceContain }: Dialo
       <ThemeProvider>
         <PopoverBoundaryProvider value={popoverRef.current} updateTrigger={updatePopover}>
           <div ref={popoverRef}>
-            <FullScreenWrapper enabled={pageCentered} onClick={closeOnBackgroundClick}>
+            <FullScreenWrapper
+              enabled={pageCentered}
+              onClick={closeOnBackgroundClick}
+              ref={fullScreenWrapperRef}
+              animationType={context.options?.animationType}
```

Comment:
Should this be a boolean - `fade={context.options?.animationType === DialogAnimationType.FADE}`?